Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -812,6 +947,12 @@ define([
                     this.dynamicScreenSpaceErrorDensity = tileset.dynamicScreenSpaceErrorDensity;
                     this.dynamicScreenSpaceErrorFactor = tileset.dynamicScreenSpaceErrorFactor;
                     this.colorBlendMode = tileset.colorBlendMode;
+                    this.skipLevelOfDetail = tileset.skipLevelOfDetail;
+                    this.skipScreenSpaceErrorFactor = tileset.skipScreenSpaceErrorFactor;
+                    this.baseScreenSpaceError = tileset.baseScreenSpaceError;
+                    this.skipLevels = tileset.skipLevels;
+                    this.immediatelyLoadDesiredLOD = tileset.immediatelyLoadDesiredLevelOfDetail;
```

Comment:
Rename `immediatelyLoadDesiredLOD` to `immediatelyLoadDesiredLevelOfDetail` to match the tileset property's name.